tests/unit_tests/test_surface_composite.py

```diff
@@ -152,6 +153,67 @@ def test_cone_one_sided(axis, point_pos, point_neg, ll_true):
     repr(s)
 
 
+@pytest.mark.parametrize(
+    "axis, indices", [
+        ("X", [0, 1, 2]),
+        ("Y", [1, 2, 0]),
+        ("Z", [2, 0, 1]),
+    ]
+)
+def test_cylinder_sector(axis, indices):
+    center = [0., 0.]
+    r1, r2 = 0.5, 1.5
+    d = (r2 - r1) / 2
+    theta0 = -60.
+    theta = 120.
+    s = openmc.model.CylinderSector(center, r1, r2, theta0, theta,
+                                    axis=axis.lower())
+    assert isinstance(s.outer_cyl, getattr(openmc, axis + "Cylinder"))
+    assert isinstance(s.inner_cyl, getattr(openmc, axis + "Cylinder"))
+    assert isinstance(s.plane0, openmc.Plane)
+    assert isinstance(s.plane1, openmc.Plane)
+
+    # Make sure boundary condition propagates
+    s.boundary_type = 'reflective'
+    assert s.boundary_type == 'reflective'
+    assert s.outer_cyl.boundary_type == 'reflective'
+    assert s.inner_cyl.boundary_type == 'reflective'
+    assert s.plane0.boundary_type == 'reflective'
+    assert s.plane1.boundary_type == 'reflective'
+
+    # Check bounding box
+    ll, ur = (+s).bounding_box
+    assert np.all(np.isinf(ll))
+    assert np.all(np.isinf(ur))
+    ll, ur = (-s).bounding_box
+    assert ll == pytest.approx(np.roll([-np.inf, -r2, -r2], indices[0]))
+    assert ur == pytest.approx(np.roll([np.inf, r2, r2], indices[0]))
+
+    # __contains__ on associated half-spaces
+    point_pos = np.roll([0, r2 + 1, 0], indices[0])
+    assert point_pos in +s
+    assert point_pos not in -s
+    point_neg = np.roll([0, r1 + d/2, r1 + d/2], indices[0])
+    assert point_neg in -s
+    assert point_neg not in +s
+
+    # translate method
+    t = uniform(-5.0, 5.0)
+    s_t = s.translate((t, t, t))
+    ll_t, ur_t = (-s_t).bounding_box
+    assert ur_t == pytest.approx(ur + t)
+    assert ll_t == pytest.approx(ll + t)
+
+    # Check invalid r1, r2 combinations
+    with pytest.raises(ValueError):
+        openmc.model.IsogonalOctagon(center, r1=1.0, r2=10.)
+    with pytest.raises(ValueError):
+        openmc.model.IsogonalOctagon(center, r1=10., r2=1.)
```

Comment:
The first check doesn't look applicable, and second one should be fixed
```suggestion
    with pytest.raises(ValueError):
        openmc.model.CylinderSector(center, r1=10., r2=1.)
```

Reply:
Whoops, good catch! Can you tell which test function I used to build the one for `CylinderSector`?